tests/performance/add_distinct_to_in_clause.xml

```diff
@@ -0,0 +1,59 @@
+<test>
+    <settings>
+        <allow_experimental_analyzer>1</allow_experimental_analyzer>
+        <prefer_localhost_replica>1</prefer_localhost_replica>
```

Comment:
Why is it `1`? I think it should be `0`, because you want to measure time for sending the temporary table.